apps/backend/test/integration/pages/user-modal.page.ts

```diff
@@ -0,0 +1,23 @@
+import {Page} from 'puppeteer';
+import {UpdateUserDto} from 'src/users/dto/update-user.dto';
+
+export class UserModalPage {
+  async userMenu(page: Page): Promise<void> {
+    await page.waitForSelector('#dropdown');
+    await page.click('#dropdown');
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Can you wait for a selector to appear on the page here instead? Sleeping in order to wait for things to show up is generally very unreliable.